config/gazebo9_bionic.yaml

```diff
@@ -0,0 +1,6 @@
+name: gazebo9bionic
+method: https://launchpad.net/~j-rivero/+archive/ubuntu/gazebo9bionic
+suites: [bionic]
+component: main
+architectures: [source]
```

Comment:
Do you think it makes sense to import the binary packages as well in order to ensure that we're soundly using packages generated from this source package? That's what we did for pyside2 and I think it makes sense to do here as well.